### PR TITLE
Feature/add co2 airvelocity

### PR DESCRIPTION
**Pull request by nbarlowATI**
WIP adding tables and code for CO2 and air velocity sensors.

### CODE REVIEW

**Comment by nbarlowATI** [comment]
Still to be done:  ingress function(s).   Probably something else.

File: __app__/crop/structure.py [hunk 1]
```diff
@@ -384,6 +386,55 @@ class ReadingsAranetTRHClass(BASE):
...
+    # columns
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    sensor_id = Column(
+        Integer,
+        ForeignKey("{}.{}".format(SENSOR_TABLE_NAME, ID_COL_NAME)),
+        nullable=False,
+    )
+
+    timestamp = Column(DateTime, nullable=False)
+    current = Column(Float, nullable=False) # raw current, in Amps
```
Comment (mhauru) [inline __app__/crop/structure.py:428]: Might we want to leave current be nullable? It seems like an optional bit of data to me, e.g. I could imagine a slightly different air velocity sensor that didn't report it.

**Comment by mhauru** [comment]
Took a quick look at what's there currently, looks really good.

File: __app__/crop/structure.py (hunk 1 above)
Comment (nbarlowATI) [reply __app__/crop/structure.py]: yep, fair enough, changed this to nullable.

File: __app__/crop/ingress_hyper.py [hunk 2]
```diff
@@ -114,34 +155,35 @@ def _get_sensor_id(aranet_pro_id, engine):
     return sensor_id
 
 
-def import_hyper_data(conn_string, database, dt_from, dt_to):
+def import_hyper_metric(engine, dt_from, dt_to, ReadingsClass, columns, sensor_type, conn_string):
```
Comment (mhauru) [inline __app__/crop/ingress_hyper.py:158]: Do we need both engine and conn_string?

**Review by mhauru — APPROVED** [review]
And with some actual proper testing too, good stuff

**Pull request merged**